### DISCUSSION TITLE
Snappy not enable

### DISCUSSION THREAD

**Discussion by eduardo-fp-romao** [confluentinc/confluent-kafka-go#896] [Q&A]
Hi guys! 

First of all thanks for the lib, great work!

I don't know if is a setting on my side or a bug but I can't use compression.type as snappy, always get this msg:
![image](https://user-images.githubusercontent.com/101641564/201317608-bc56818c-a5a9-4d7f-af09-0686f43248e5.png)

Can u guys help me?

Thanks

**Comment by edenhill** [comment]
What's your librdkafka version? (LibraryVersion())

**Reply by eduardo-fp-romao** [reply]
![image](https://user-images.githubusercontent.com/101641564/201323079-8db984f1-a65e-42e5-9c24-7b83d5a74065.png)
1.9.2

**Reply by edenhill** [reply]
Print the output from `kafka.LibraryVersion()`

**Reply by eduardo-fp-romao** [reply]
![image](https://user-images.githubusercontent.com/101641564/201327206-8a07869c-a36c-4f12-a6f8-d808fb368e0e.png)
![image](https://user-images.githubusercontent.com/101641564/201327047-26c72b0f-a2e3-4fbb-ac7b-af109afd5353.png)
17367807 - 1.9.2

**Reply by edenhill** [reply]
Okay, are you building your go application with `-tags dynamic`?
What OS and architecture are you on?
Have you installed librdkafka separately?

**Reply by eduardo-fp-romao** [reply]
1. I'm not using -tags dynamic, when I run `go run .` in local get this error (sry I'm new to go, don't know if this answer)
2. Using windows
3. And I don't have the lib installed

**Reply by edenhill** [reply]
Found the issue, will fix in the upcoming 2.0 release (a month out).

**Reply by edenhill** [reply]
https://github.com/edenhill/librdkafka/issues/4063

**Reply by eduardo-fp-romao** [reply]
thanks :)

**Reply by edenhill** [reply]
Actually, I didn't find it, it doesn't make sense because WITH_SNAPPY is defined. Back to zero

**Reply by eduardo-fp-romao** [reply]
So what could cause this issue? 
![image](https://user-images.githubusercontent.com/101641564/202504905-83458913-ac61-4910-9a81-ba54d98d7939.png)

**Comment by edenhill** [comment]
What operating system are you on?
How are you building your Go program? (any `-tags ..`?)
